StratisCore.UI/src/app/wallet/status-bar/status-bar.component.html

```diff
@@ -1,22 +1,12 @@
 <div class="status" *ngIf="(generalInfo | async) as _generalInfo">
   <ul class="list-inline align-self-center row">
-    <li *ngIf="!(percentSynced == '100%')" class="list-inline-item mx-1" placement="top" ngbTooltip="{{toolTip}}">
-      <i class="lnr lnr-cloud-sync text-warning"></i>
-      <small class="text-warning">{{ percentSynced }}</small>
+    <li class="list-inline-item mx-1" placement="top" ngbTooltip="{{toolTip}}" tooltipClass="status-tooltip">
+      <i class="lnr lnr-cloud" style="font-size: 140%"></i>
+      <small>{{ percentSynced }}</small>
     </li>
-    <li *ngIf="percentSynced == '100%'" class="list-inline-item mx-1" placement="top" ngbTooltip="{{toolTip}}">
-      <i class="lnr lnr-cloud-check text-success"></i>
-      <small class="text-success">{{ percentSynced }}</small>
-    </li>
-    <li class="list-inline-item mx-1" placement="top" ngbTooltip="{{ connectedNodesTooltip }}">
-      <i class="lnr lnr-earth text-black"></i>
-      <small class="text-black">{{ _generalInfo.connectedNodes }}</small>
-    </li>
-    <li *ngIf="!stakingService.stakingEnabled && !globalService.getSidechainEnabled()" class="list-inline-item mx-1" placement="top" ngbTooltip="Not staking">
-      <i class="lnr lnr-lock text-danger"></i>
-    </li>
-    <li *ngIf="stakingService.stakingEnabled && !globalService.getSidechainEnabled()" class="list-inline-item mx-1" placement="top" ngbTooltip="Staking">
-      <i class="lnr lnr-lock text-success"></i>
+    <li class="list-inline-item mx-1" placement="top" ngbTooltip="{{ connectedNodesTooltip }}" tooltipClass="status-tooltip">
+      <i class="lnr lnr-earth"></i>
+      <small>{{ _generalInfo.connectedNodes }}</small>
```

Comment:
We should still display if staking is active or not. Many users just have a quick glance here to check if they're still staking.